src/biokbase/auth.py

```diff
@@ -41,6 +41,48 @@ def __init__(self, info_dict: dict, token: str = None):
         self.token = token if token is not None else info_dict.get("token")
 
 
+class UserRole:
+    def __init__(self, role_info):
+        self.role_id = role_info.get("id")
+        self.description = role_info.get("desc")
+
+
+class PolicyId:
+    def __init__(self, policy_info):
+        self.policy_id = policy_info.get("id")
+        self.agree_date = policy_info.get("agreedon")
```

Comment:
_The least feared of the local mafia bosses was Frankie Deltorino, better known as "the Agree Don"._